events/janus_gelfevh.c

```diff
@@ -164,14 +164,14 @@ static int janus_gelfevh_connect(void) {
 				janus_network_address_to_string_buffer(&addr, &addr_buf) != 0) {
 		if(res)
 			freeaddrinfo(res);
-		JANUS_LOG(LOG_ERR, "Could not resolve address (%s): %d (%s)\n", backend, errno, strerror(errno));
+		JANUS_LOG(LOG_ERR, "Could not resolve address (%s): %d (%s)\n", backend, errno, g_strerror(errno));
```

Comment:
Here if `freeaddrinfo` returns `EAI_SYSTEM` it will set errno, and otherwise its error code should be passed to `gai_strerror`. I think that kind of change belongs in a separate PR though.